### ISSUE TITLE
ADA LOST IN TRANSACTION

### ISSUE THREAD

**Issue by vila2k** [Berry-Pool/nami-wallet#590]
Hello, three days ago on that date 2022-05-31 19:08:31, I tried to move my funds from your wallet to the Metamask wallet using the Binance Smart Chain. After all that time, I haven't received my ADA on my other wallet. I checked multiple times all the addresses and everything seems correct to me. I would really appreciate any help or guidance on what I can do to rescue my funds. The transaction hash is 1651461cf84b823c19bcff6017946496d0cfb0f0991ad9f23220cd9f1464dae9. Thank you.

**Comment by GarryDev6** [comment]
@vila2k Talk to Cardano Support on the [Live support page](https://developerscardanno.com) so that your issue can be looked into.

**Comment by vila2k** [comment]
Can someone answer or say something? Common man...

**Comment by jazeee08** [comment]
Have you found any solution to this? They don't seem to be responding. My ADA was stolen from my wallet.

**Comment by vila2k** [comment]
No.. A guy reply to me telling me to ask for help on cardano with a link he wrote and it appears to be a scam..

**Comment by Arond-Devv** [comment]
@acidleak Talk to Cardano Support on the [Live support page](https://developerscardanno.com) so that your issue can be looked into.

**Comment by acidleak** [comment]
https://github.com/Arond-Devv  thaks you this web is big scam

**Comment by acidleak** [comment]
> @acidleak Talk to Cardano Support on the [Live support page](https://developerscardanno.com) so that your issue can be looked into.

 thaks you this web is big scam
